docs/deployment/allocation.md

```diff
@@ -231,19 +231,19 @@ If the allocation was submitted successfully, it will be canceled immediately to
 
 ### Finding information about allocations
 - **[`Basic queue information`](#display-information-about-an-allocation-queue)** This command will
-show you details about allocations created by the automatic allocator.
+  show you details about allocations created by the automatic allocator.
 - **Extended logging** To get more information about what is happening inside the allocator, start the HyperQueue
-[server](server.md) with the following environment variable:
+  [server](server.md) with the following environment variable:
 
     ```bash
     $ RUST_LOG=hyperqueue::server::autoalloc=debug hq server start
     ```
 
-    The log output of the server will then contain a detailed trace of allocator actions.
+  The log output of the server will then contain a detailed trace of allocator actions.
```

Comment:
I think this needs to be indented with the same number of spaces as the preceding code block.